pkgs/by-name/qo/qownnotes/package.nix

```diff
@@ -41,11 +42,13 @@ stdenv.mkDerivation (finalAttrs: {
     qt6Packages.qtsvg
     qt6Packages.qtwebsockets
     botan3
+    libgit2
   ] ++ lib.optionals stdenv.hostPlatform.isLinux [ qt6Packages.qtwayland ];
 
   cmakeFlags = [
     "-DQON_QT6_BUILD=ON"
     "-DBUILD_WITH_SYSTEM_BOTAN=ON"
+    "-DBUILD_WITH_LIBGIT2=ON"
```

Comment:
Don't we want to make this feature optional until it is no longer experimental?

Reply:
I shouldn't have called it "experimental", because it really isn't...

Reply:
I'll change the changelog.